procfile/README.md

```diff
@@ -0,0 +1,52 @@
+# Procfile
+
+This is a Go library for parsing the Procfile format.
+
+## Formats
+
+### Standard
+
+The standard Procfile format is what you're probably most familiar with, which maps a process name to the command to run. An example of a standard Procfile might look like:
+
+```yaml
+web: ./bin/web
+worker: ./bin/worker
+```
+
+The standard Procfile format is specified in https://devcenter.heroku.com/articles/procfile.
+
+### Extended
+
+The extended Procfile format is Empire specific and implements a subset of the attributes defined in the [docker-compose.yml](https://docs.docker.com/compose/yml/) format. The extended Procfile format gives you more control, and allows you to configure things like health checks for the individual processes. An example of an extended Procfile might look like:
+
+```yaml
+web:
```

Comment:
On a side note, we should move PORT allocation into the ECS implementation of scheduler.Scheduler, since it's specific to ECS.